modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -215,7 +218,9 @@ The following example configures Anypoint Connector for Database (Database Conne
 [[configure-plugin-dependencies]]
 == Configure Additional Plugin Dependencies
 
-You can configure additional plugin dependencies for the connectors in your application. +
+You can configure additional plugin dependencies for a connector in your application, which makes the dependency available only for that connector. This configuration enables you to configure different dependencies, or even multiple versions of the same dependency, for different connectors. +
```

Comment:
Be sure to standardize to either "dependency" or "dependencies" in the first sentence.
Just an alternative:
```suggestion
Configuring additional plugin dependencies for a connector in your application makes those dependencies available for only that connector. Doing this enables you to configure different dependencies, or even multiple versions of the same dependency, for different connectors. +
```